### DISCUSSION TITLE
Inline mode for RainbowKit

### DISCUSSION THREAD

**Discussion by omarsy** [rainbow-me/rainbowkit#1395] [Ideas]
I've noticed that the RainbowKit package currently only offers modal mode for displaying UI elements. I think it would be extremely useful and beneficial to also have an inline mode available.

Inline mode would allow elements to be displayed directly in the page's content flow, without opening a separate modal window. This would allow for smoother integration and greater flexibility when using the package in different scenarios and layouts.

**Comment by DanielSinclair** [comment]
@omarsy This isn't currently planned. You can always rely on Wagmi's connectors directly for a more custom workflow like that; this is what many of the large dApps have done.
